### DISCUSSION TITLE
Allocation tables for decompositions

### DISCUSSION THREAD

**Discussion by Terraffin** [eclipse-syson/syson#896] [Ideas]
Loving how the tool is developing! I was wondering if allocation tables as below would be supported?

![allocation tables](https://pivotpt.com/.res/images/mbse/sysml/SysML-Allocation-Table-A.png)

E.g, function architecture on the left, logical architecture on the top, with arrows in the intersections to show allocation of functions to logical subsystems.

**Comment by AxelRICHARD** [comment]
Hello,

Thank you for your interest in SysON!

We have not planned yet such table representation but it is definitely something that will be useful in the tool!

Regards,
